app/controllers/collaborators_controller.rb

```diff
@@ -0,0 +1,42 @@
+class CollaboratorsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @wiki = Wiki.find(params[:wiki_id])
+    @collaborator = Collaborator.new
+  end
+
+  def create
+    @wiki = Wiki.find(params[:wiki_id])
+    @collaborator = Collaborator.new(collaborator_params)
+    @collaborator.wiki = @wiki
+
```

Comment:
@bjblock - How do I assign the current user to the wiki as a collaborator here? I feel like I'm close but I don't have the code that actually assigns the current user as a collaborator.